src/CodeExamples.sln

```diff
@@ -23,6 +23,40 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMap", "HashMap\HashMap.
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TestHashMap", "TestHashMap\TestHashMap.csproj", "{EBDC072A-4291-4482-9BAE-2490C190B1B6}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.CalculateUintCapacity", "HashMap.Test\HashMapNamespace.CalculateUintCapacity\HashMapNamespace.CalculateUintCapacity.csproj", "{4DC0AECB-322A-427A-B49B-FA54AF5A8D16}"
+EndProject
+# Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.CalculateCapacity", "HashMap.Test\HashMapNamespace.CalculateCapacity\HashMapNamespace.CalculateCapacity.csproj", "{52CF5877-0FA0-482F-A9E8-137C1FF052EC}"
+# EndProject
+# Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.ProvideCapacity", "HashMap.Test\HashMapNamespace.ProvideCapacity\HashMapNamespace.ProvideCapacity.csproj", "{6B3046FE-22DE-43B3-9335-83883D159BA1}"
+# EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.Add", "HashMap.Test\HashMapNamespace.Add\HashMapNamespace.Add.csproj", "{8513B597-4E9C-44CD-809F-290F446E21E4}"
+EndProject
+# Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.AddOrUpdate", "HashMap.Test\HashMapNamespace.AddOrUpdate\HashMapNamespace.AddOrUpdate.csproj", "{DD451542-3BE6-47B2-8D29-B614A0EE2F6B}"
+# EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.Remove", "HashMap.Test\HashMapNamespace.Remove\HashMapNamespace.Remove.csproj", "{2FB8A7E4-D031-466B-8834-DA4DA8265808}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.ContainsKey", "HashMap.Test\HashMapNamespace.ContainsKey\HashMapNamespace.ContainsKey.csproj", "{49821606-F17E-4E4A-A1D0-5F5D052120F9}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.TryGetValue", "HashMap.Test\HashMapNamespace.TryGetValue\HashMapNamespace.TryGetValue.csproj", "{E252454D-7F25-45DC-9CF1-C6DB354EB67D}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.Clear", "HashMap.Test\HashMapNamespace.Clear\HashMapNamespace.Clear.csproj", "{C54A2E51-5AAA-42BE-BF3D-C5BF7FD9C557}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.GetEnumerator", "HashMap.Test\HashMapNamespace.GetEnumerator\HashMapNamespace.GetEnumerator.csproj", "{CB575984-6982-4487-8944-B4751E0970F0}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.HashMap64Node", "HashMap.Test\HashMapNamespace.HashMap64Node\HashMapNamespace.HashMap64Node.csproj", "{1ABBB916-2C9C-4558-95EE-EDDCEBB4A8A1}"
+EndProject
+# Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "HashMapNamespace.ToString", "HashMap.Test\HashMapNamespace.ToString\HashMapNamespace.ToString.csproj", "{393CAC49-92C5-44D5-9019-96179864A0FF}"
+# EndProject
+# Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "RealtimePlugin.Run", "RealtimePlugin.Test\RealtimePlugin.Run\RealtimePlugin.Run.csproj", "{75DCAF84-C05A-4CFD-A0EF-A03426D009D4}"
```

Comment:
### Compilation Error
```
vscode ➜ .../CodeExamples/src/RealtimePlugin.Test/RealtimePlugin.Run (roost-1709036480) $ dotnet test
  Determining projects to restore...
  Restored /workspaces/CodeExamples/src/PluginInterface/PluginInterface.csproj (in 2.03 sec).
  Restored /workspaces/CodeExamples/src/RealtimePlugin/RealtimePlugin.csproj (in 2.03 sec).
  Restored /workspaces/CodeExamples/src/RealtimePlugin.Test/RealtimePlugin.Run/RealtimePlugin.Run.csproj (in 2.04 sec).
  PluginInterface -> /workspaces/CodeExamples/src/PluginInterface/bin/Debug/net6.0/PluginInterface.dll
  RealtimePlugin -> /workspaces/CodeExamples/src/Plugins/Debug/net6.0/RealtimePlugin.dll
/workspaces/CodeExamples/src/RealtimePlugin.Test/RealtimePlugin.Run/Run.cs(97,57): error CS0246: The type or namespace name 'SimpleIocAdapter' could not be found (are you missing a using directive or an assembly reference?) [/workspaces/CodeExamples/src/RealtimePlugin.Test/RealtimePlugin.Run/RealtimePlugin.Run.csproj]
/workspaces/CodeExamples/src/RealtimePlugin.Test/RealtimePlugin.Run/Run.cs(104,65): error CS0117: 'State' does not contain a definition for 'Unavailable' [/workspaces/CodeExamples/src/RealtimePlugin.Test/RealtimePlugin.Run/RealtimePlugin.Run.csproj]
/workspaces/CodeExamples/src/RealtimePlugin.Test/RealtimePlugin.Run/Run.cs(116,13): warning NUnit2005: Consider using the constraint model, Assert.That(actual, Is.EqualTo(expected)), instead of the classic model, Assert.AreEqual(expected, actual) (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit2005.md) [/workspaces/CodeExamples/src/RealtimePlugin.Test/RealtimePlugin.Run/RealtimePlugin.Run.csproj]
/workspaces/CodeExamples/src/RealtimePlugin.Test/RealtimePlugin.Run/Run.cs(126,13): warning NUnit2005: Consider using the constraint model, Assert.That(actual, Is.EqualTo(expected)), instead of the classic model, Assert.AreEqual(expected, actual) (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit2005.md) [/workspaces/CodeExamples/src/RealtimePlugin.Test/RealtimePlugin.Run/RealtimePlugin.Run.csproj]
/workspaces/CodeExamples/src/RealtimePlugin.Test/RealtimePlugin.Run/Run.cs(136,13): warning NUnit2005: Consider using the constraint model, Assert.That(actual, Is.EqualTo(expected)), instead of the classic model, Assert.AreEqual(expected, actual) (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit2005.md) [/workspaces/CodeExamples/src/RealtimePlugin.Test/RealtimePlugin.Run/RealtimePlugin.Run.csproj]
/workspaces/CodeExamples/src/RealtimePlugin.Test/RealtimePlugin.Run/Run.cs(159,13): warning NUnit2005: Consider using the constraint model, Assert.That(actual, Is.EqualTo(expected)), instead of the classic model, Assert.AreEqual(expected, actual) (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit2005.md) [/workspaces/CodeExamples/src/RealtimePlugin.Test/RealtimePlugin.Run/RealtimePlugin.Run.csproj]
/workspaces/CodeExamples/src/RealtimePlugin.Test/RealtimePlugin.Run/Run.cs(149,13): warning NUnit2005: Consider using the constraint model, Assert.That(actual, Is.EqualTo(expected)), instead of the classic model, Assert.AreEqual(expected, actual) (https://github.com/nunit/nunit.analyzers/tree/master/documentation/NUnit2005.md) [/workspaces/CodeExamples/src/RealtimePlugin.Test/RealtimePlugin.Run/RealtimePlugin.Run.csproj]
vscode ➜ .../CodeExamples/src/RealtimePlugin.Test/RealtimePlugin.Run (roost-1709036480) $ 
```